backend/modules/sse/views.py

```
@@ -0,0 +1,53 @@
+from django.http import HttpResponse
+from drf_spectacular.utils import extend_schema
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from modules.sse import live_prices
+from modules.sse.schemas import SSERequestParams
+from modules.sse.serializers import SSERequestSerializer
+
+
+class SSESubscribeView(APIView):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)